coordinator/.env.example

```diff
@@ -1,2 +1,22 @@
+# Rate limit configuation
 TTL=60000
 LIMIT=10
+
+# Make sure your private and public RSA keys are generated (see package.json scripts)
+# Public key must be copied to frontend app to encrypt user sensitive data
+COORDINATOR_PUBLIC_KEY_PATH="./pub.key"
+COORDINATOR_PRIVATE_KEY_PATH="./priv.key"
+
+# Make sure you have TallyVotesQv.zkey, TallyVotesQv.wasm
+# and TallyVotesNonQv.zkey, TallyVotesNonQv.wasm inside COORDINATOR_ZKEY_PATH folder
+# https://maci.pse.dev/docs/trusted-setup
+COORDINATOR_TALLY_ZKEY_NAME=TallyVotes
+
+# Make sure you have ProcessMessagesQv.zkey, ProcessMessagesQv.wasm 
+# and ProcessMessagesNonQv.zkey, ProcessMessagesNonQv.wasm inside COORDINATOR_ZKEY_PATH folder
```

Comment:
I think we should consider adding variables related to `rapidsnark` and c++ witness gen. Running rounds with the WASM witness gens is gonna take 100x more times to generate proofs